staging/src/k8s.io/api/policy/v1beta1/types.go

```diff
@@ -77,8 +77,43 @@ type PodDisruptionBudgetStatus struct {
 
 	// total number of pods counted by this disruption budget
 	ExpectedPods int32 `json:"expectedPods" protobuf:"varint,6,opt,name=expectedPods"`
+
+	// Conditions contain conditions for PDB. The disruption controller sets the
+	// DisruptionAllowed condition, which will have one of the following three reasons:
+	// - SyncFailed: The controller encountered an error and wasn't able to compute
+	//               the number of allowed disruptions. Therefore no disruptions are
+	//               allowed and the status of the condition will be false.
+	// - InsufficientPods: The number of pods are either at or below the number
+	//                     required by the PodDisruptionBudget. No disrutiptions are
+	//                     allowed and the status of the condition will be false.
+	// - SufficientPods: There are more pods than required by the PodDisruptionBudget.
+	//                   The condition will be true, and the number of allowed
+	//                   disruptions are provided by the DisruptionsAllowed property.
```

Comment:
use the serialized name for the field:
```suggestion
	//                   disruptions are provided by the disruptionsAllowed property.
```